project/AppDependencies.scala

```diff
@@ -2,15 +2,15 @@ import sbt._
 
 object AppDependencies {
 
-  val silencerVersion = "1.7.0"
+  val silencerVersion = "1.7.1"
 
   val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc" %% "play-partials" % "8.1.0-play-27",
-    "uk.gov.hmrc" %% "url-builder" % "3.5.0-play-27",
+    "uk.gov.hmrc" %% "play-partials" % "8.1.0-play-28",
+    "uk.gov.hmrc" %% "url-builder" % "3.5.0-play-28",
     "uk.gov.hmrc" %% "tax-year" % "1.3.0",
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-27" % "5.3.0",
-    "uk.gov.hmrc" %% "play-frontend-hmrc" % "0.68.0-play-27",
-    "uk.gov.hmrc" %% "http-caching-client" % "9.4.0-play-27",
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "5.4.0",
+    "uk.gov.hmrc" %% "play-frontend-hmrc" % "0.81.0-play-28",
+    "uk.gov.hmrc" %% "http-caching-client" % "9.5.0-play-27",
```

Comment:
upgrade **http-caching-client** to play-28

Reply:
Done